frontend/src/app/commonComponents/Checkboxes.tsx

```diff
@@ -24,6 +24,7 @@ interface Props {
   validationStatus?: "error" | "success";
   onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
   required?: boolean;
+  inputRef?: React.RefObject<HTMLInputElement>;
```

Comment:
It might be nicer to do this with `React.forwardRef` instead of having a unique prop name, since I would guess we'll end up wanting this for the other wrapped form components as well at some point.